test/runtime/includes/lookup.test.ts

```diff
@@ -83,3 +83,11 @@ test("doesn't cache falsy messages", () => {
     pt: { field_2: 'nome' },
   });
 });
+
+test('updates message of a locale dictionary', () => {
+  addMessages('en', { field: 'name' });
+  expect(lookup('field', 'en')).toBe('name');
+
+  addMessages('en', { field: 'name2' });
+  expect(lookup('field', 'en')).toBe('name2');
+});
```

Comment:
This test name doesn't seem to be related to what we're actually testing. Maybe:

```suggestion
test('clears a locale lookup cache when new messages are added', () => {
  addMessages('en', { field: 'name' });
  expect(lookup('field', 'en')).toBe('name');

  addMessages('en', { field: 'name2' });
  expect(lookup('field', 'en')).toBe('name2');
});
```